air/src/farewell_step/outcome.rs

```diff
@@ -100,7 +113,8 @@ fn populate_outcome_from_contexts(
 }
 
 /// Deduplicate values in a supplied vector.
-fn dedup<T: Eq + Hash>(mut vec: Vec<T>) -> Vec<T> {
+#[tracing::instrument]
```

Comment:
I don't need we actually need to instrument it, other functions will dominate over this deduplication.